packages/cli/src/commands/apps.ts

```diff
@@ -29,6 +29,15 @@ export default class AppsCommand extends APICommand<typeof AppsCommand.flags> {
 		description: 'the app id or number from list',
 	}]
 
+	static examples = [
+		{ description: 'list all apps', command: 'smartthings apps' },
+		{ description: 'list the first app in the list retrieved by running "smartthings apps"', command: 'smartthings apps 1' },
+		{ description: 'list an app by id', command: 'smartthings apps <app-id>' },
+		{ description: 'include URLs and ARNs in the output', command: 'smartthings apps --verbose' },
+		{ description: 'list only SERVICE classification devices', command: 'smartthings apps --classification SERVICE' },
+		{ description: 'list only API only apps', command: 'smartthings apps --type API_ONLY' },
```

Comment:
The description might be a little easier to parse if it were 'list only API-only apps'

Reply:
Good observation. This also wasn't supposed to be in the PR. I've fixed this PR to not include it at all and I'll update in PR #422.

Reply:
Fixed in #422 now.